### PR TITLE
autoid_service: add unit test for the package

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve?

Issue Number: close None
Problem Summary:

### What is changed and how it works?

Add some unit test 

### Check List

Tests 

- [X] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: meta/autoid/autoid.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/opentracing/opentracing-go"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
-	autoid "github.com/pingcap/tidb/autoid_service"
+	"github.com/pingcap/kvproto/pkg/autoid"
```
Comment (Benjamin2037) [inline meta/autoid/autoid.go:29]: So this mod， also have same problem that without test filei int this folder.

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**